lib/real_page/credentials.rb

```
@@ -0,0 +1,12 @@
+module RealPage
+  # Encapsulation of data needed to authenticate with RealPage
+  class Credentials
+    attr_reader :username, :password, :license_key
```

If you leave any of these out, the request will fail. Since we aren't persisting these outside of the running process, I'm happy with letting the request fail loudly for now. Eventually, this class will go away in favor or setting these values in the configuration.